pkg/reconciler/openshift/tektonconfig/rbac.go

```
@@ -399,9 +428,8 @@ func (r *rbac) createRoleBinding(ctx context.Context, sa *corev1.ServiceAccount)
 		Subjects: []rbacv1.Subject{{Kind: rbacv1.ServiceAccountKind, Name: sa.Name, Namespace: sa.Namespace}},
 	}
 
-	_, err = rbacClient.RoleBindings(sa.Namespace).Create(ctx, rb, metav1.CreateOptions{})
-	if err != nil {
-		logger.Error(err, "creation of 'edit' rolebinding failed, in Namespace", sa.GetNamespace())
+	if _, err := rbacClient.ClusterRoleBindings().Create(ctx, crb, metav1.CreateOptions{}); err != nil {
```

Should we return an error here?

---

Yes 
Updated :+1: